references/validationFunctions/specimen/tumourGrade.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Validates that tumour_grade is a permissable value based on tumour_grading_system
+ * 
+ * @param {Object} $row The object representing the row for a donor. Object keys represent the fields.
+ * @param {String} $field The value for the field.
+ */
+const validation = ($row,$field) => 
+    (function validate() {
+        let result = {valid: true, message: "Ok"};
+        let codeList = [];
+        switch ($row.tumour_grading_system.trim().toLowerCase()) {
+            case 'default':
```

Comment:
😆 was so confused as to why `default` case was first, but this is of course the string value.

Reply:
this tripped me out too lol